Formula/k/kertish-dfs.rb

```diff
@@ -2,9 +2,9 @@ class KertishDfs < Formula
   desc "Kertish FileSystem and Cluster Administration CLI"
   homepage "https://github.com/freakmaxi/kertish-dfs"
   url "https://github.com/freakmaxi/kertish-dfs/archive/refs/tags/v22.2.0147.tar.gz"
-  version "22.2.0147-532592"
```

Comment:
This is the version that upstream binaries show, so we probably shouldn't remove it:
```
❯ curl -OL https://github.com/freakmaxi/kertish-dfs/releases/download/v22.2.0147/osx_v22.2.0147_arm64.zip
[snip]
❯ unzip osx_v22.2.0147_arm64.zip
[snip]
❯ ./krtfs version
Kertish-dfs (v22.2.0147-532592) usage:
[snip]
```